examples/api-samples/src/browser/monaco-editor-preferences/monaco-editor-preference-extractor.ts

```diff
@@ -83,6 +83,8 @@ export class MonacoEditorPreferenceSchemaExtractor implements CommandContributio
     @inject(MessageService) protected readonly messageService: MessageService;
     @inject(FileService) protected readonly fileService: FileService;
 
+    protected lastPreferenceName: string;
```

Comment:
Since we don't really have a lot of context while we are in the `withLocalization` method, we need to store the current preference name somewhere to generate the translation key:

https://github.com/eclipse-theia/theia/blob/3047e761145895418608084da7242cdcc3abf8b0/examples/api-samples/src/browser/monaco-editor-preferences/monaco-editor-preference-extractor.ts#L173